sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -387,6 +414,10 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 	}
 
 	return links;
+
+devm_err:
+	dev_err(dev, "memory allocated failed\n");
```

Comment:
@bardliao we typically don't add such messages, maintainers don't like them.

Reply:
@plbossart done.